// Copyright 2016 Google Inc.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//    http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

// Defines a simple protocol buffers used in the test of the EqualsProto
// matcher.

syntax = "proto3";

package exegesis.testing;

message SubProto {
  int32 field_a = 1;
  int32 field_b = 2;
}

message TestProto {
  int32 integer_field = 1;
  string string_field = 2;
  SubProto message_field = 3;
}
